### PR TITLE
Fix docs for visualization -- fixes #845

### LINKED ISSUE #845: Add necessary imports to Docs for monitoring example
Need to update the example config in https://parsl.readthedocs.io/en/stable/userguide/monitoring.html to include necessary imports.

### PR DESCRIPTION
**Pull request by ZhuozhaoLi**
fixes #845 and modify docs for using visualization server

### CODE REVIEW

File: docs/userguide/monitoring.rst [hunk 1]
```diff
@@ -50,6 +59,8 @@ Install the visualization server::
    $ pip install git+https://github.com/Parsl/viz_server.git
 
 Once `viz_server` is installed, you can run the utility `parsl-visualize` in the directory with the
-monitoring.db sqlite file to launch a web page with the workflow visualization.
+`monitoring.db` sqlite file to launch a web page for the workflow visualization::
+
+   $ parsl-visualize sqlite:///<absolute-full-path-to-db>
```
Comment (yadudoc) [inline docs/userguide/monitoring.rst]: We should point out the absurd 4 slashes necessary in a note. Eg: `sqlite:////tmp/monitoring.db`.
Absolute-path is sufficient, we don't need to say absolute-full-path

File: docs/userguide/monitoring.rst [hunk 2]
```diff
@@ -23,11 +23,20 @@ Here's an example configuration that logs monitoring information to a local sqli
...
+    from parsl.addresses import address_by_hostname
+
+    import logging
+
     config = Config(
         executors=[
             HighThroughputExecutor(
                 label="local_htex",
                 cores_per_worker=1,
+                max_workers=4,
```
Comment (yadudoc) [inline docs/userguide/monitoring.rst]: Aren't we missing the monitoring config here ? Say include something like : 
```python
threads_config = Config(
    executors=[
	HighThroughputExecutor(
            label="local_htex",
            cores_per_worker=1,
            address=address_by_hostname(),
        )
    ],
    monitoring=MonitoringHub(
        hub_address=address_by_hostname(),
        hub_port=55055,
        logging_level=logging.INFO,
        resource_monitoring_interval=10,
    ),
    strategy=None
)
```

Comment (ZhuozhaoLi) [reply docs/userguide/monitoring.rst]: It is already there. You can see it if you view the whole file.

File: docs/userguide/monitoring.rst (hunk 1 above)
Comment (ZhuozhaoLi) [reply docs/userguide/monitoring.rst]: added new lines to explain that

**Pull request merged**